lib/tasks/referral_method_and_hire_stats_export.rake

```diff
@@ -24,11 +24,11 @@ namespace :referral_method_and_hire_stats do
         row = []
         row << ar.created_at.to_date.to_s
         row << ar.user.full_name
-        row << REFERRAL_METHODS.select do |rm|
+        row << (REFERRAL_METHODS.select do |rm|
           ar.data.find do |prompt, response, _data_type, _id|
             prompt == rm && response.try(:downcase) == 'yes'
           end.present?
-        end.try(:join, ', ') || 'None given'
+        end.join(', ').presence || 'None given')
```

Comment:
It is the same because `join` _does_ work on empty arrays and `any_array.select { anything }` will never return `nil` — the `try` was meaningless here as was the `||` because `[]` is truthy.

Actually, Rubocop's complaint about operator precedence exposed that this code... never worked. That is, they had written:

```ruby
row << thing || 'None given'
```

but it turns out that `<<` has higher precedence than `||` so that was the equivalent of

```ruby
(row << thing) || 'None given'
```

The `'None given'` was additionally meaningless because even if `join` returned a falsey value, it wouldn't end up in the row.